expo/expo
# [web] [image] Fixed not accepting props in expo-image like data props

# Why


I couldn't add extra props like data-style in expo-image web.

# How


I have spread all the extra props that user passes which were earlier not there.

# Test Plan


1. Pass the `data-style` to image example in bare-expo application. 
2. Inspect the image on web.
3. See if those props have been added to the rendered image.

# Checklist
